internal/worker/bootstrap/worker.go

```diff
@@ -178,16 +183,28 @@ func (w *bootstrapWorker) loop() error {
 	if err != nil {
 		return errors.Annotatef(err, "getting bootstrap params")
 	}
-	bootstrapArgs, err := w.seedControllerCharm(ctx, dataDir, args)
+	controllerConfig, err := w.cfg.ControllerConfigService.ControllerConfig(ctx)
 	if err != nil {
 		return errors.Trace(err)
 	}
+	if err := w.seedControllerCharm(ctx, dataDir, args, controllerConfig); err != nil {
+		return errors.Trace(err)
+	}
 
 	// Retrieve bootstrap addresses for setting API host ports.
-	_, err = w.getBoostrapAddresses(ctx, bootstrapArgs.BootstrapMachineInstanceId)
+	bootstrapAddresses, err := w.getBoostrapAddresses(ctx, args.BootstrapMachineInstanceId)
```

Comment:
Can you fix the `getBootstrapAddresses` typo.

Reply:
no longer needed, this function has been removed on your refactor